templates/10-Operational Analysis/operational-process.html.j2

```diff
@@ -0,0 +1,35 @@
+{#
+    Copyright DB InfraGO AG and contributors
+    SPDX-License-Identifier: Apache-2.0
+#}
+{%
+  from 'common_macros.html.j2' import
+  get_cap_type,
+  render_functional_chain,
+  display_traceability,
+  show_other_attributes
+%}
+{%- set cap_type = get_cap_type(object) -%}
```

Comment:
This call should happen inside of `render_functional_chain`, since that macro is the only party interested in the result.